musicbot/bot.py

```diff
@@ -1109,9 +1109,15 @@ def run(self):
 
     def _gen_embed(self):
         """Provides a basic template for embeds"""
+        if not self.config.footer_text:
+            text = 'Just-Some-Bots/MusicBot ({})'.format(BOTVERSION)
```

Comment:
Why not just set the default in config.py to this rather than doing this conditional?

Reply:
done